src/test/java/org/ehrbase/fhirbridge/fhir/observation/BodyHeightIT.java

```diff
@@ -0,0 +1,138 @@
+package org.ehrbase.fhirbridge.fhir.observation;
+
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import org.ehrbase.fhirbridge.comparators.CustomTemporalAcessorComparator;
+import org.ehrbase.fhirbridge.ehr.converter.BodyHeightCompositionConverter;
+import org.ehrbase.fhirbridge.ehr.converter.radiologischerBefund.RadiologischerBefundConverter;
+import org.ehrbase.fhirbridge.ehr.opt.korpergrossecomposition.KorpergrosseComposition;
+import org.ehrbase.fhirbridge.ehr.opt.korpergrossecomposition.definition.GrosseLangeObservation;
+import org.ehrbase.fhirbridge.fhir.AbstractMappingTestSetupIT;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.DiagnosticReport;
+import org.hl7.fhir.r4.model.Observation;
+import org.javers.core.Javers;
+import org.javers.core.JaversBuilder;
+import org.javers.core.diff.Diff;
+import org.javers.core.metamodel.clazz.ValueObjectDefinition;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.temporal.TemporalAccessor;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BodyHeightIT extends AbstractMappingTestSetupIT {
+
+    public BodyHeightIT() {
+        super("Observation/BodyHeight/", Observation.class); //fhir-Resource
+    }
+
+    @Test
+    void createBodyHeightLoincPeriod() throws IOException {
+        create("create-body-height-loinc-period.json");
+    }
+
+    @Test
+    void createBodyHeightSnomedPeriod() throws IOException {
+        create("create-body-height-snomed-period.json");
+    }
+
+    // #####################################################################################
+    // check payload
+    @Test
+    void mapping_normal() throws IOException {
```

Comment:
i changed something here, now a testMapping method is to be overwritten. Take a look on the develop branch.

Reply:
Jep, much better code now :)